traceroute/traceroute.c

```diff
@@ -784,9 +784,14 @@ static void print_probe (probe *pb) {
 }
 
 
-static void print_end (void) {
+static void print_end (int result) {
 
-	printf ("\n");
+	if (result)
```

Comment:
please try to match the existing indentation style in the file

Reply:
the file indentation is inconsistent, let's merge it